pyof/foundation/network_types.py

```diff
@@ -201,7 +201,7 @@ def __init__(self, tlv_type=127, value=BinaryData()):
         """
         super().__init__()
         self.tlv_type = tlv_type
-        self._value = value
+        self._value = BinaryData() if value is None else value
```

Comment:
The cleaner way is to set `BinaryData()` directly in the `__init__` method - same behavior, better readability.